### ISSUE TITLE
Better-looking default skin

### ISSUE THREAD

**Issue by recapitalverb** [swoolcock/rush#35]
IMO the hitobjects should be in a similar style to osu!lazer's skin.

The regular minion could look something like this (rough sketch done in Krita):
![Screenshot_20200706_232232](https://user-images.githubusercontent.com/41869184/86616919-b5173d00-bfe0-11ea-9ee1-0acada3890a6.png)

**Comment by frenzibyte** [comment]
Yeah the minions we have are just placeholders as we care for the mechanics of the game more than the style currently.

@swoolcock The design looks nice to me, but would like to take your opinion into account as well.

**Comment by swoolcock** [comment]
Would need to change the player sprite then too, otherwise it would look weird.

**Comment by swoolcock** [comment]
@frenzibyte @recapitalverb any suggestions on the player sprite then?  You have no idea how long I spent looking for a free asset that didn't look like total ass.

**Comment by recapitalverb** [comment]
I'll try sketching out some ideas I have soon.

**Comment by recapitalverb** [comment]
My artistic skills apparently disappointed me, so I opted for something simpler; I think we could go for the stickman seen in the Rush! logo in README.md: 
![image](https://user-images.githubusercontent.com/41869184/86758667-379a0e00-c06e-11ea-900c-861122ecc500.png)

**Comment by recapitalverb** [comment]
To implement the above in the game, it's probably okay to store animations in monochrome and then use that as a mask for an instance of  `osu.Game.Graphics.Backgrounds.Triangles`, though I'm not very familiar with osu!framework.

**Comment by swoolcock** [comment]
@recapitalverb do you have the file for that triangle minion you could share?

**Comment by recapitalverb** [comment]
It was a mere sketch so I didn't have the project file, sorry. Though you could just use `osu.Game.Graphics.Backgrounds.Triangles` and a clipping container with outline and glow, as mentioned above. Building from components already in osu!lazer and !framework are better here anyways.

**Comment by swoolcock** [comment]
Yeah, I'm not sure how simple it is to make rounded corners on a triangle though.  I'll fiddle with it a bit and see what I can do.
